usl_pipeline/cloud_functions/main_test.py

```diff
@@ -123,6 +124,97 @@ def test_build_feature_matrix(mock_storage_client, mock_firestore_client):
     )
 
 
+@mock.patch.object(main.error_reporting, "Client", autospec=True)
+@mock.patch.object(main.firestore, "Client", autospec=True)
+@mock.patch.object(main.storage, "Client", autospec=True)
+def test_build_feature_matrix_wrf(
+    mock_storage_client,
+    mock_firestore_client,
+    mock_error_reporting_client,
+):
+    # Get some random data to place in a netcdf file
+    netcdf_array = numpy.array([[1, 2, 3], [4, 5, 6], [7, 8, 9]], dtype=numpy.float32)
+
+    # Build a netcdf file
+    with netCDF4.Dataset("met_em_test.nc", mode="w", format="NETCDF4") as ncfile:
```

Comment:
It looks like this is writing a file named 'met_em_test.nc' to the file system. Can we use a BytesIO buffer instead? Or at least a tempfile?

Reply:
Thanks - updated this so we're working with an in-memory file